core/templates/pages/blog-dashboard-page/blog-dashboard-page.component.spec.ts

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for Blog Dashboard page component.
+ */
+
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ComponentFixture, fakeAsync, TestBed, tick, waitForAsync } from '@angular/core/testing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { CapitalizePipe } from 'filters/string-utility-filters/capitalize.pipe';
+import { MockTranslatePipe } from 'tests/unit-test-utils';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import { LoaderService } from 'services/loader.service';
+import { AlertsService } from 'services/alerts.service';
+import { BlogDashboardPageComponent } from './blog-dashboard-page.component';
+import { BlogDashboardBackendApiService } from 'domain/blog/blog-dashboard-backend-api.service';
+import { MatTabsModule } from '@angular/material/tabs';
+
+class MockCapitalizePipe {
+  transform(input: string): string {
+    return input;
+  }
+}
+
```

Comment:
If you are using this in multiple places, then move this mock pipe to `unit-test-utils.ts` and export it from there.

Reply:
Moved to unit-test-utils.